Libraries/xcsdk/Headers/xcsdk/SDK/Manager.h

```diff
@@ -88,7 +89,7 @@ class Manager {
     /*
      * Load from a developer root. Returns nullptr on error.
      */
-    static std::shared_ptr<Manager> Open(libutil::Filesystem const *filesystem, std::string const &path);
+    static std::shared_ptr<Manager> Open(config::Config const *config, libutil::Filesystem const *filesystem, std::string const &path);
```

Comment:
I've been told by an Oculus engineer that general styling is always to use references unless you absolutely need to use a pointer :)